### DISCUSSION TITLE
table custom pagination and on_pagination_change seems to not working together

### DISCUSSION THREAD

**Discussion by Asterix45** [zauberzeug/nicegui#3241] [Q&A]
### Question

Hi all,

I'm trying to use pagination and get __on_pagination_change__ events, but seem that the two configuration are not working together.
I'm with nicegui 1.4.26 and I use this code

```python
columnsDef = [
    {'name': 'id',       'label': 'ID',       'field': 'id',       'required': True, 'sortable': True},
    {'name': 'username', 'label': 'Username', 'field': 'username', 'required': True, 'sortable': True},
    {'name': 'password', 'label': 'Password', 'field': 'password', 'required': True, 'sortable': False},
]

table_data= {
    'rows': [],
    'pagination': {
        'rowsPerPage': 10,
        'descending': True,
        'sortBy': 'id',
        'page': 0,
        'rowsNumber': 0,
    },
}

@ui.refreshable
def getTable():
    table = ui.table(
        columns=columnsDef,
        rows=table_data['rows'],
        pagination=table_data['pagination'],
        on_pagination_change=lambda e: ui.notify(e.value)
    )
```

with a button I add some rows in the `table_data['rows']` list, put the right params in the `table_data['pagination']` dict and call the `getTable.refresh()` .
The table is refreshed with the new data but the events are not notified as __on_pagination_change__.
if I avoid the specific pagination configuration and put only this

```python
...

@ui.refreshable
def getTable():
    table = ui.table(
        columns=columnsDef,
        rows=table_data['rows'],
        pagination=1,
        on_pagination_change=lambda e: ui.notify(e.value)
    )
```

the __on_pagination_change__ events are notified correctly. 
Is there a configuration/way to have pagination configuration, with page and count, and event signal?

**Comment by falkoschindler** [comment]
Hi @Asterix45,

Please share an executable example including all relevant parts, i.e. the button you're describing. This allows us and the community to help more efficiently. Thanks!
[Why is it important to provide a minimal reproducible example?](https://github.com/zauberzeug/nicegui/wiki/FAQs#why-is-it-important-to-provide-a-minimal-reproducible-example)
